ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml

```diff
@@ -0,0 +1,245 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 0 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: redhat-developer
+    repo: gitops-operator
+  labels:
+    ci-operator.openshift.io/variant: v4.12
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-developer-gitops-operator-master-v4.12-periodic-kuttl-parallel
```

Comment:
I think we should add notification for our channel if the tests failed, WDYT?

Reply:
Sounds good! I'll take a look on how I could implement it